tests/integration/kubernetes/setup.sh

```diff
@@ -44,24 +44,22 @@ setup_policy_files() {
 	create_common_genpolicy_settings "${workloads_work_dir}"
 }
 
-add_kernel_initrd_annotations_to_yaml() {
+add_annotations_to_yaml() {
 	local yaml_file="$1"
-	local mariner_kernel_path="/usr/share/cloud-hypervisor/vmlinux.bin"
-	local mariner_initrd_path="/opt/kata/share/kata-containers/kata-containers-initrd-mariner.img"
+	local annotation_name="$2"
+	local annotation_value="$3"
 	local resource_kind="$(yq read ${yaml_file} kind)"
 
 	case "${resource_kind}" in
 
 	Pod)
 		echo "Adding kernel and initrd annotations to ${resource_kind} from ${yaml_file}"
-		yq write -i "${K8S_TEST_YAML}" 'metadata.annotations[io.katacontainers.config.hypervisor.kernel]' "${mariner_kernel_path}"
-		yq write -i "${K8S_TEST_YAML}" 'metadata.annotations[io.katacontainers.config.hypervisor.initrd]' "${mariner_initrd_path}"
+		yq write -i "${K8S_TEST_YAML}" 'metadata.annotations[${annotation_name}]' "${annotation_value}"
```

Comment:
@GabyCT, I think we have to use double quotes "" here instead of single ones '', otherwise ${annotation_value} won't be expanded.